chapter13/sxg/ex_13_1.pl

```diff
@@ -0,0 +1,23 @@
+#!/usr/bin/perl
+use warnings;
+use strict;
+
+print "Enter directory structure: ";
+chomp( my $dir = <STDIN> );
+if ( $dir eq "" ) {
+	chdir or die "Can't chdir to home:  $!";
+}
+else {
+	chdir $dir or die "Can't chdir to '$dir': $!";
+}
+
+&print_file_from_current_dir();

```

Comment:
Why do you need a subroutine here when it's already going to execute the code below here anyway?